test/byContract/IexecPocoBoost/IexecPocoBoost.test.ts

```diff
@@ -2414,19 +2421,19 @@ function addressToBytes32(address: string): string {
  * @param volume number of tasks in the deal
  * @returns amount of total stake
  */
-function computeSchedulerDealStake(workerpoolPrice: number, volume: number) {
-    return ((workerpoolPrice * workerpoolStakeRatio) / 100) * volume;
+function computeSchedulerDealStake(workerpoolPrice: bigint, volume: bigint) {
+    return (workerpoolPrice * workerpoolStakeRatio * volume) / 100n;
```

Comment:
Careful the order is important here.
```suggestion
    // Order is important here.
    return ((workerpoolPrice * workerpoolStakeRatio) / 100n) * volume;
```